packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -99,6 +101,8 @@ install -m 0644 %{S:4} %{buildroot}%{_cross_templatedir}/kubelet-kubeconfig
 install -m 0644 %{S:5} %{buildroot}%{_cross_templatedir}/kubernetes-ca-crt
 install -m 0644 %{S:6} %{buildroot}%{_cross_templatedir}/kubelet-exec-start-conf
 install -m 0644 %{S:7} %{buildroot}%{_cross_templatedir}/kubelet-bootstrap-kubeconfig
+install -m 0644 %{S:11} %{buildroot}%{_cross_templatedir}/kubernetes-tls-crt
+install -m 0644 %{S:12} %{buildroot}%{_cross_templatedir}/kubernetes-tls-private-crt
```

Comment:
You might also consider rendering both the certificate and the key to a single templated file, e.g. `/etc/kubernetes/pki/kubelet-server-current.pem`, the way that the combined client / server files are today under `/var/lib/kubelet/pki`.